.github/workflows/individual-image-scanner-quay.yaml

```diff
@@ -0,0 +1,159 @@
+---
+name: Fetch security scan results from quay
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  scans:
+    runs-on: ubuntu-latest
+    outputs:
+      as-output: ${{ steps.access-setup-scan.outputs.VULNERABILITIES_EXIST }}
+      cs-output: ${{ steps.cluster-setup-scan.outputs.VULNERABILITIES_EXIST }}
+      gd-output: ${{ steps.gateway-deployment-scan.outputs.VULNERABILITIES_EXIST }}
+      kr-output: ${{ steps.kcp-registrar-scan.outputs.VULNERABILITIES_EXIST }}
+      sc-output: ${{ steps.shellcheck-scan.outputs.VULNERABILITIES_EXIST }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            access-setup:
+            - 'images/access-setup/**'
+            cluster-setup:
+              - 'images/cluster-setup/**'
+            gateway-deployment:
+              - 'images/gateway-deployment/**'
+            kcp-registrar:
+              - 'images/kcp-registrar/**'
+            shellcheck:
+              - 'test/images/shellcheck/**'
+
+      - name: access-setup scans
+        continue-on-error: true
+        id: access-setup-scan
+        if: steps.filter.outputs.access-setup == 'true'
+        run: |
+          ./test/images/clair-scan/scan-image.sh
+          echo "::set-output name=VULNERABILITIES_EXIST::$(./test/images/clair-scan/scan-image.sh | tail -1)"
```

Comment:
Aren't you running the scan-image.sh script twice? 
If that's the case, I suggest:
```
./test/images/clair-scan/scan-image.sh | tee /tmp/clair-scan.log
echo "::set-output name=VULNERABILITIES_EXIST::$(tail -1 /tmp/clair-scan.log)"
```
That way you still have the output in the job for the user, and can access them in the job without needing to re-run the script.